### PR TITLE
PR for Issue Dap-24

### PR DESCRIPTION
**Pull request by jamesleemeredith**

### CODE REVIEW

File: Section09/9.2-Tableau.md [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+---
+## AGGREGATION
+- The process of converting a set of values into a single value (ex. Sum, Average, Median)
+- Done based on Measures and Dimensions
+- Aggregating Measures can be added to relational data sources
+- Aggregating Dimensions can be done by Minimum, Maximum, Count or Count(Distinct)
+
+<br>
+
+### Let’s Try it Using the Sample Superstore Data:
```
Comment (Danielle0727) [inline Section09/9.2-Tableau.md:11]: Suggestion on formatting, however, this is my own personal style I like to use so please use your better judgment on what you like best. For items you want them to click on, you could put them in brackets and bold them. For example, "Drag [**Sales**] to "Rows", ..."

File: Section09/9.2-Tableau.md [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+    - Date and Time Values​
+    - Boolean Values
+    - Geographic Values
+- Can be manually assigned from the preview pane by clicking on the icon at the top of each column, or via the "Fields" pane on the left-hand side
+
+<br>
+
+---
+## DATA ROLES
+### Dimensions vs. Measures​
```
Comment (Danielle0727) [inline Section09/9.2-Tableau.md:64]: Maybe add that Dimensions are in green and Measures are in blue. That helped me a lot when I was a student and it pointed out.

File: Section09/9.2-Tableau.md [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+- Measures - produce an axes when added to row/column, numeric​
+### Discrete vs. Continuous​ Data
+- Every field is categorized as either discrete or continuous​
+
+<br>
+
+Discrete Data: Notice how every quantity has a header at the bottom
+<br>
+
+![Discrete Data Types in Tableau](images\Discrete_data_graph.png)
```
Comment (Danielle0727) [inline Section09/9.2-Tableau.md:76]: The links say "Data not found". May have to remove the "!". I had the same thing happen with one of mine and removing the exclamation point fixed it. I think "!" is only used with images.

File: Section09/9.2-Tableau.md [hunk 4]
```diff
@@ -0,0 +1,367 @@
...
+
+<br>
+
+---
+## WORKING WITH DEFAULT FIELDS
+### Adding a comment to a field:
+- You can view a comment when you put your cursor over the field​
+- Right-click on a field > select "Default Properties"​ > select "Comment​"
+
+### Changing a whole number field to a decimal (and vis versa):
```
Comment (Danielle0727) [inline Section09/9.2-Tableau.md:102]: Vice Versa typo.

File: Section09/9.2-Tableau.md [hunk 5]
```diff
@@ -0,0 +1,367 @@
...
+- Right-click on a field, > select "Default Properties​" > select "Number Format​" > select "Number"​ > set "Decimal Places" to the desired value
+### Changing The Fiscal Year:
+- Right click on a field​ > select "Default Properties"​ > "Fiscal Year Start"​ > select desired month from the drop-down menu
+
+<br>
+
+---
+## CREATING ALIASES
+- Individual data points and be given aliases in Tableau
```
Comment (Danielle0727) [inline Section09/9.2-Tableau.md:111]: May want to remove "be".

File: Section09/9.2-Tableau.md [hunk 6]
```diff
@@ -0,0 +1,367 @@
...
+- Second Class would be where you would rename
+
+<br>
+
+---
+# FILTERING DATA
+## DIMENSION FILTERS
+- Allow you to filter data by dimension type
+- Types of dimension filters:
+    - General: selects one or multiple members from the domain​
```
Comment (Danielle0727) [inline Section09/9.2-Tableau.md:123]: May help the reader to bold **General"", ""Wildcard**, **Condition**, and **Top**

File: Section09/9.2-Tableau.md [hunk 7]
```diff
@@ -0,0 +1,367 @@
...
+- Create a filter​
+- You can remove the filter by dragging Sub-Category outside of the "Filters" pane
+
+<br>
+
+---
+
+## MEASURE FILTERS
+- Allow you to filter measure data types
+- Types of measure filters:
```
Comment (Danielle0727) [inline Section09/9.2-Tableau.md:146]: May help to bold these as well: **Range of Values**, **At Least**, **At Most**, **Special**

File: Section09/9.2-Tableau.md [hunk 8]
```diff
@@ -0,0 +1,367 @@
...
+- You can remove the filter by dragging Sales outside of the "Filters" pane
+
+<br>
+
+---
+
+## DATE FILTERS
+- Allow you to filter your data by specified date range
+-Types of date filters:
+    - Relative Dates: can specify a range of dates that are updated based on the date and time
```
Comment (Danielle0727) [inline Section09/9.2-Tableau.md:173]: If you choose to bold, I included the markdown for ease of copying and pasting: **Relative Dates**, **Range of Dates**, **Discrete Dates**, **Individual Dates**, **Additional Date Filter Options**

File: Section09/9.2-Tableau.md [hunk 9]
```diff
@@ -0,0 +1,367 @@
...
+- COMBINED sets - Two sets combined to compare different members​
+- BUILDING sets - Created through manual selection, condition, ranking
+
+<br>
+
+---
+## BINS
+- Equal-sized containers that store data values that correspond to or fit within the bin size
+- Can use any discrete field to create them​
+- Created by right-clicking on a field and selecting "Create" > "Bins​"
```
Comment (Danielle0727) [inline Section09/9.2-Tableau.md:300]: If you wanted to include an actual arrow pointing right, here is the markdown code: 

&rarr;

File: Section09/9.2-Tableau.md [hunk 10]
```diff
@@ -0,0 +1,367 @@
...
+- Example: Category & Sub-Category in Rows and SUM(Sales) in Text​
+- Can be used as dimensions​
+- Created by right-clicking a field, selecting "Hierarchy" > "Create Hierarchy", and then adding dimensions as desired
+
+<br>
+
+---
+# CREATING CHARTS
+- Charts are often easier to understand then tables with numbers​, and are usually more visually appealing
+- Options for charts and be selecting by clicking on the “Show Me” button on the upper right corner​ of the task bar
```
Comment (Danielle0727) [inline Section09/9.2-Tableau.md:317]: I wasn't sure if "be selecting" was meant to say "by selecting"

File: Section09/9.2-Tableau.md (hunk 5 above)
Comment (JohanBester) [reply Section09/9.2-Tableau.md:111]: I think you wanted to write ...
"Individual data points can be given aliases in Tableau"

File: Section09/9.2-Tableau.md (hunk 10 above)
Comment (JohanBester) [reply Section09/9.2-Tableau.md:317]: I think you meant to say ...
"Charts are often easier to understand than tables with numbers​,"

**Review by JohanBester — CHANGES_REQUESTED** [review]
James I reviewed your PR and made a few inline comments.

Apart from those and what I communicated with you on Slack, I have one last style suggestion:

You might want to add a small little intro to the top of the section, and a small summary to the bottom of the section

File: Section09/9.2-Tableau.md [hunk 11]
```diff
@@ -0,0 +1,446 @@
...
+- MUST pivot from Columns to Rows (tall vs. wide)
+
+### To create a pivot
+
+- Select all desired columns in the Preview Pane
+- Right-click and select [**Pivot**] from the pop-up menu
+
+<br>
+
+## The Data interpreter
```
Comment (HollymRobinson) [inline Section09/9.2-Tableau.md:42]: Suggestion: ## The Data Interpreter

File: Section09/9.2-Tableau.md [hunk 12]
```diff
@@ -0,0 +1,446 @@
...
+- You can change the default aggregation of any measure by clicking on a measure within the [**Data**] pane (located on the far left-hand side), select [**Default Properties**] > [**Aggregations**] and then select a desired aggregation
+
+<br>
+
+## PIVOTS
+
+- Allow you to select columns that need to be manipulated and transform them into a row for data manipulation
+- MUST pivot from Columns to Rows (tall vs. wide)
+
+### To create a pivot
```
Comment (HollymRobinson) [inline Section09/9.2-Tableau.md:35]: Suggestion: ### To Create a Pivot

File: Section09/9.2-Tableau.md [hunk 13]
```diff
@@ -0,0 +1,446 @@
...
+- Shows trend of data over a period of time​
+- Normally used when there’s a date data type
+- Drag [**Order Date**] into [**Columns**]
+- Drag [**Sales**] into [**Rows**] (Marks type should change automatically to line)​
+- Date can be discrete or continuous (Right-click on Order Date)
+
+---
+
+## SUMMARY
+
```
Comment (HollymRobinson) [inline Section09/9.2-Tableau.md:445]: Good Job! I will add bolding to my markdowns for consistency.

**Review by HollymRobinson — COMMENTED** [review]
Only suggestion two minor edits. Good job!

File: Section09/9.2-Tableau.md [hunk 14]
```diff
@@ -0,0 +1,446 @@
...
+<br>
+
+Continuous Data: Notice how quantity values are shown in a continuous axis
+
+![Continuous Data Types in Tableau](./images/Continuous_data_graph.png)
+
+<br>
+
+---
```
Comment (HollymRobinson) [inline Section09/9.2-Tableau.md:109]: copy pasted from lines 66 to 109

**Review by JohanBester — APPROVED** [review]
Approved. Let's merge it!  :)

**Pull request merged**